tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -12279,6 +12283,10 @@ void hlsl::HandleDeclAttributeForHLSL(Sema &S, Decl *D, const AttributeList &A,
     declAttr = ::new (S.Context) HLSLRayPayloadAttr(
         A.getRange(), S.Context, A.getAttributeSpellingListIndex());
     break;
+  /*case AttributeList::AT_HLSLCXXOverload:
+    declAttr = ::new (S.Context) HLSLCXXOverloadAttr(
+        A.getRange(), S.Context, A.getAttributeSpellingListIndex());
+    break;*/
```

Comment:
Why is this commented case here?  Should it be removed (not accessible from HLSL directly)?

Reply:
Removed. I was strugging to figure out why the attribute wasn't propagating as expected and this was my (incorrect) thought.